scripts/get-helm-3

```diff
@@ -255,7 +255,7 @@ fail_trap() {
 # testVersion tests the installed client to make sure it is working.
 testVersion() {
   set +e
-  HELM="$(command -v $BINARY_NAME)"
+  HELM="$(command -v $HELM_INSTALL_DIR/$BINARY_NAME)"
```

Comment:
I think this is (unfortunately) too simple. I think we need to conditionally check:

- Iff the user sets `HELM_INSTALL_DIR`, validate the installed binary is executable at `$HELM_INSTALL_DIR/$BINARY_NAME` ie. `command -v $HELM_INSTALL_DIR/$BINARY_NAME`
- Else, if the user does not set `HELM_INSTALL_DIR`, check `$BINARY_NAME`, which would be expected to be in the user's path. ie. the current test: `command -v $BINARY_NAME`

I think the current point of `testVersion` is to verify that Helm is accessible to the user in their PATH (see the error message). This change allows this check to succeed, even if that isn't true.

Reply:
You can see from the script, if the user does not set `HELM_INSTALL_DIR`,   `HELM_INSTALL_DIR` will be default value "/usr/local/bin" , so `command -v $HELM_INSTALL_DIR/$BINARY_NAME` is fine.
```
: ${HELM_INSTALL_DIR:="/usr/local/bin"}
```
I think the verification method for 'testVersion' should also be consistent with the following methods `checkHelmInstalledVersion`.
```
checkHelmInstalledVersion() {
  if [[ -f "${HELM_INSTALL_DIR}/${BINARY_NAME}" ]]; then
    local version=$("${HELM_INSTALL_DIR}/${BINARY_NAME}" version --template="{{ .Version }}")
    if [[ "$version" == "$TAG" ]]; then
      echo "Helm ${version} is already ${DESIRED_VERSION:-latest}"
      return 0
    else
      echo "Helm ${TAG} is available. Changing from version ${version}."
      return 1
    fi
  else
    return 1
  fi
}
```